extensions/amp-sidebar/validator-amp-sidebar.protoascii

```diff
@@ -93,5 +93,6 @@ tags: {  # amp-sidebar > nav
   child_tags: {
     mandatory_num_child_tags: 1
     child_tag_name_oneof: "UL"
+    child_tag_name_oneof: "OL"
```

Comment:
I think the CSS for `.i-amphtml-toolbar > ul` (in `amp-sidebar.css`) need to be updated as well.

maybe we can add two new `visual-tests` for `toolbar`? One for `ul` and one for `ol`?

Reply:
Sgtm, working on adding visual tests now.